src/Module/Util/VaInfo.php

```diff
@@ -757,7 +760,8 @@ private function get_metadata_order()
             'getid3_tag_order' => static::getConfigContainer()->get(ConfigurationKeyEnum::GETID3_TAG_ORDER)
         ];
 
-        return (array) ($tagorderMap[$this->get_metadata_order_key()] ?? '');
+        // convert to lower case to be sure it matches plugin names in Ampache\Plugin\PluginEnum
+        return array_map('strtolower', $tagorderMap[$this->get_metadata_order_key()] ?? '');
```

Comment:
Any chance for multibyte characters? mb_strtolower would be better then

Reply:
there isn't now and i think unlikely will so not a big deal